Patch: fix_admin.py.patch
Author: Riccardo Setti <giskard@debian.org>
Submitted upstream: http://sourceforge.net/tracker/index.php?func=detail&aid=1532081&group_id=103&atid=300103
Improve the search method in the admin page wich list members (Closes: #359721)
Index: mailman-2.1.10/Mailman/Cgi/admin.py
===================================================================
--- mailman-2.1.10.orig/Mailman/Cgi/admin.py	2008-06-15 18:49:55.000000000 +0200
+++ mailman-2.1.10/Mailman/Cgi/admin.py	2008-06-15 18:49:58.000000000 +0200
@@ -950,7 +950,7 @@
         for letter in digits + lowercase:
             if not buckets.get(letter):
                 continue
-            url = adminurl + '/members?letter=%s' % letter
+            url = adminurl + '/members?findmember=%s&letter=%s' %(urllib.quote(regexp) ,letter)
             if letter == bucket:
                 show = Bold('[%s]' % letter.upper()).Format()
             else:
